pages/admin/request/create-new.tsx

```diff
@@ -0,0 +1,845 @@
+          toast({
+            status: 'success',
+            description: 'New application has been submitted!',
+            isClosable: true,
+          });
+
+          if (applicationId) {
+            router.push(`/admin/request/${applicationId}`);
+          }
+        }
+      }
+    },
+    onError: error => {
+      toast({
+        status: 'error',
+        description: error.message,
+        isClosable: true,
+      });
+    },
+  });
+
+  /**
+   * Handle new APP request submission
+   */
+  const handleSubmit = async (event: SyntheticEvent) => {
+    event.preventDefault();
+
+    if (physicianAssessment.patientCondition === null) {
+      toast({ status: 'error', description: 'Missing patient condition', isClosable: true });
+      return;
+    }
+
+    if (physicianAssessment.permitType === null) {
+      toast({ status: 'error', description: 'Missing permit type', isClosable: true });
+      return;
+    }
+
+    if (additionalQuestions.usesAccessibleConvertedVan === null) {
+      toast({
+        status: 'error',
+        description: 'Missing if patient uses accessible converted van',
+        isClosable: true,
+      });
+      return;
+    }
+
+    if (additionalQuestions.requiresWiderParkingSpace === null) {
+      toast({
+        status: 'error',
+        description: 'Missing if patient requires wider parking space',
+        isClosable: true,
+      });
+      return;
+    }
+
+    if (doctorInformation.mspNumber === null) {
+      toast({ status: 'error', description: 'Missing physician MSP number', isClosable: true });
+      return;
+    }
+
+    if (paymentDetails.paymentMethod === null) {
+      toast({ status: 'error', description: 'Missing payment method', isClosable: true });
+      return;
+    }
+
+    await submitNewApplication({
+      variables: {
+        input: {
+          firstName: permitHolderInformation.firstName,
+          middleName: permitHolderInformation.middleName,
+          lastName: permitHolderInformation.lastName,
+          email: permitHolderInformation.email,
+          phone: permitHolderInformation.phone,
+          receiveEmailUpdates: permitHolderInformation.receiveEmailUpdates,
+          addressLine1: permitHolderInformation.addressLine1,
+          addressLine2: permitHolderInformation.addressLine2,
+          city: permitHolderInformation.city,
+          postalCode: permitHolderInformation.postalCode,
+          dateOfBirth: permitHolderInformation.dateOfBirth,
+          gender: permitHolderInformation.gender,
+          otherGender: permitHolderInformation.otherGender,
+
+          disability: physicianAssessment.disability,
+          disabilityCertificationDate: physicianAssessment.physicianCertificationDate,
+          patientCondition: physicianAssessment.patientCondition,
+          mobilityAids: null, //TODO: get mobility aids when forms are updated to get this data
+          otherPatientCondition: physicianAssessment.patientEligibilityDescription || null,
```

Comment:
These fields actually have updated names in `NewApplication`. In the `PhysicianAssessment` type, there is a TODO to replace the field names. Could we update those hardcoded field names to use the picked fields from `NewApplication` instead?